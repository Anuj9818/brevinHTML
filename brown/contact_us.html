<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta name="description" content="">
<meta name="author" content="">
<link rel="shortcut icon" href="images/favicon.png" type="image/x-icon">
<title>Contact us |  Brown</title>
<link rel="stylesheet" type="text/css" href="css/bootstrap.css" />
<link rel="stylesheet" type="text/css" href="font-awesome-4.7.0/css/font-awesome-4.7.0.min.css">
<link rel="stylesheet" type="text/css" href="css/style.css" />
<link rel="stylesheet" type="text/css" href="css/responsive.css" />

<!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
<!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
<!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->

<link href="https://fonts.googleapis.com/css?family=Josefin+Sans:400,600,700" rel="stylesheet">
<link href="https://fonts.googleapis.com/css?family=Lato:400,400i,700,700i" rel="stylesheet">
</head>

<body>
<div id="billboard">
  <header class="header"><!--//.header starts-->
    <div id="pav-mainnav">
      <div class="navigation-bar">
        <div class="container">
          <div class="navbar" role="navigation">
            <div class="navbar-header navbar-inverse">
              <figure class="logo_holder"> <a href="index.html"><img src="images/main-logo.png"></a> </figure>
              <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar"></span> <span class="icon-bar"></span> <span class="icon-bar"></span> </button>
            </div>
            <!--navbar-inverse-->
            <div class="collapse navbar-collapse">
              <ul class="nav navbar-nav">
                <li><a href="index.html">Home </a></li>
                <li><a href="about_us.html"> About Us </a></li>
                <li><a href="services.html"> Services </a> </li>
                <li class="active"><a href="contact_us.html" > Contact Us</a></li>
              </ul>
              <!--navbar-nav--> 
            </div>
            <!-- /.nav-collapse --> 
          </div>
          <!--navbar--> 
        </div>
      </div>
      <!--navigation-bar--> 
    </div>
    <!--pav-mainnav--> 
    
  </header>
  <!--//.header ends--> 
  
</div>
<!--billboard ends--> 

<!--.//.MAIN SLIDER STARTS-->

<section class="sub_page-banner">
  <figure class="full_banner"> <img src="images/welcome_banner.jpg" alt="" title=""> </figure>
  <div class="breadCrumbNav">
    <div class="container">
      <h2 class="page_title">Contact us</h2>
    </div>
  </div>
</section>
<!--.//.MAIN SLIDER ENDS--> 

<!--//CONTACT US CONTENT STARTS-->
<section class="wrapper contact_bar">
  <div class="container">
    <div class="row">
      <div class="col-sm-6">
        <form role="form" class="contact_form">
          <div class="form-group">
            <input id="name" class="form-control" type="text" placeholder="Your Name:" title="Type your full name">
          </div>
          <div class="form-group">
            <input id="email" class="form-control" type="email" placeholder="E-Mail:" title="Type your email">
          </div>
          <div class="form-group">
            <input id="phone" class="form-control" type="text" placeholder="Phone:" title="Type your pnone no">
          </div>
          <div class="form-group">
            <input id="subject" class="form-control" type="text" placeholder="Subject:" title="Type your Subject">
          </div>
          <div class="form-group">
            <textarea id="message" class="form-control"  placeholder="Your Message..."></textarea>
          </div>
          <div class="form-group">
            <input class="btn-submit" type="submit" value="Submit" data-loading-text="Loading...">
          </div>
        </form>
      </div>
      <div class="col-sm-6">
        <div class="map">
          <div class="gmap_canvas"> 
            <!-- <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3531.9042008141796!2d85.31781121511978!3d27.72024398278663!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x39eb191b9f0ca651%3A0x720f23d97bcbcd91!2sTAG+nepal+Treks+%26+Expedition+Pvt+Ltd!5e0!3m2!1sen!2sus!4v1544172177689" width="600" height="450" frameborder="0" style="border:0" allowfullscreen></iframe>-->
            
            <figure class="gmap"> <img src="images/demo_map.png" alt="This is map"> </figure>
          </div>
        </div>
      </div>
    </div>
  </div>
</section>

<!--//CONTACT US CONTENT ENDS-->

<footer class="footer wrapper">
  <div class="container">
    <h2>Contact us</h2>
    <div class="row inner_content">
      <div class="col-sm-4 col-xs-12">
        <address>
        <figure class="icon"> <i class="fa fa-map-marker" aria-hidden="true"></i> </figure>
        <div class="details"> <span> your address </span> </div>
        </address>
      </div>
      <div class="col-sm-4 col-xs-12">
        <address>
        <figure class="icon"> <i class="fa fa-phone" aria-hidden="true"></i> </figure>
        <div class="details"> <span> <a href="tel:01-5555039" class="linkto">01-5555039 </a> <a href="tel:01-98124578" class="linkto">01-98124578</a></span> </div>
        </address>
      </div>
      <div class="col-sm-4 col-xs-12">
        <address>
        <figure class="icon"> <i class="fa fa-envelope" aria-hidden="true"></i> </figure>
        <div class="details"> <span> <a href="mailto:info@brownbeauty.com" class="linkto">info@brownbeauty.com</a> </span> </div>
        </address>
      </div>
      <div class="col-xs-12">
        <div class="social-icons">
          <ul>
            <li><a href="#" data-toggle="tooltip" data-placement="top" title="" data-original-title="Facebook"><i class="fa fa-facebook" aria-hidden="true"></i> </a></li>
            <li><a href="#" data-toggle="tooltip" data-placement="top" title="" data-original-title="Twitter"><i class="fa fa-twitter" aria-hidden="true"></i> </a></li>
            <li><a href="#" data-toggle="tooltip" data-placement="top" title="" data-original-title="Linkedin"><i class="fa fa-linkedin" aria-hidden="true"> </i></a></li>
            <li><a href="#" data-toggle="tooltip" data-placement="top" title="" data-original-title="Google-plus"><i class="fa fa-google-plus" aria-hidden="true"></i> </a></li>
          </ul>
        </div>
      </div>
    </div>
  </div>
</footer>
<section class="copyright">
  <div class="container">
    <p>The Brown Beauty lounge Â©2019. All Rights Reserved</p>
  </div>
</section>
<section class="back-top">
  <div id="back-top" class="uparrow" style="display: block;"> <a href="#top" title="Go to the top of the page"> <i class="fa fa-angle-up"></i> </a> </div>
</section>
</body>
<script src="js/jquery-1.9.1.min.js"></script>
<script src="js/fixed-nav.js"></script>
<script src="js/jquery.js"></script>
<script src="js/bootstrap.js"></script>
<script src="js/Push-up.js"></script>
<script src="js/Jquery simple slider.js"></script>

<!--..//SOCIAL ICON HOVER TOOLTIP STARTS-->
<script>
$(document).ready(function(){
    $('[data-toggle="tooltip"]').tooltip();   
});
</script>
</html>
