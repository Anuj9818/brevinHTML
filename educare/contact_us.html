<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta name="description" content="">
<meta name="author" content="">
<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
<link rel="shortcut icon" href="images/favicon.png" type="image/x-icon">
<title>Educare | contact us</title>
<link rel="stylesheet" type="text/css" href="css/bootstrap.css" />
<link rel="stylesheet" type="text/css" href="font-awesome-4.7.0/css/font-awesome-4.7.0.min.css">
<link rel="stylesheet" type="text/css" href="css/animate.css" />
<link rel="stylesheet" type="text/css" href="css/flexslider.css" />
<link rel="stylesheet" type="text/css" href="lightbox/css/lightbox.css">
<link rel="stylesheet" type="text/css" href="css/owl.carousel.css"/>
<link rel="stylesheet" type="text/css" href="SmartDroddownMenus/smart_dropdown.css">
<link rel="stylesheet" type="text/css"  href="css/reset.css"/>
<link rel="stylesheet" type="text/css" href="css/style.css" />
<link rel="stylesheet" type="text/css" href="css/navbar.css"/>
<link rel="stylesheet" type="text/css" href="css/side_nav.css" />
<link rel="stylesheet" type="text/css" href="css/responsive.css" />
<link href="https://fonts.googleapis.com/css?family=Rubik:400,500,700&display=swap" rel="stylesheet">

<!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
<!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
<!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->

</head>
<body>
<header class="header"><!--../header starts--> 
  <!-- ./top-bar-->
  <div class="topbar">
    <div class="container">
      <div class="top_lft_bar">
        <figure class="icon"> <i class="fa fa-phone" aria-hidden="true"></i> </figure>
        <div class="details"> <span>01-4464483</span> </div>
        <div class="cont_divider"></div>
        <figure class="icon"> <i class="fa fa-envelope" aria-hidden="true"></i> </figure>
        <div class="details"> <a href="mailto:info@educare.edu.np" class="linkto">info@educare.edu.np</a> </div>
      </div>
      <div class="top_rht_bar">
        <div class="top_social_icons">
          <ul>
            <li><a href="#" title=""><i class="fa fa-facebook" aria-hidden="true"></i> </a></li>
            <li><a href="#"  title=""><i class="fa fa-twitter" aria-hidden="true"></i> </a></li>
            <li><a href="#"  title=""><i class="fa fa-linkedin" aria-hidden="true"> </i></a></li>
            <li><a href="#"  title=""><i class="fa fa-youtube-play" aria-hidden="true"></i></a></li>
          </ul>
        </div>
        <span class="year">Since 2008 </span> </div>
    </div>
  </div>
  
  <!-- ./top-bar-->
  <div id="pav-mainnav"><!-- ./pav-mainnav-->
    <div class="navigation-bar"><!-- ./navigation-bar-->
      <div class="container">
        <div class="row">
          <div class="col col-sm-3 col-xs-12">
            <figure class="logo_holder bounceIn animatable"><a href="index.html"> <img src="images/main_logo.png"> </a></figure>
          </div>
          <div class="col col-sm-9 col-xs-12">
            <div id="mainContent">
              <div id="myCanvasNav" class="overlay3" onclick="closeOffcanvas()" style="width: 0%; opacity: 0;"></div>
              <div id="myOffcanvas" class="offcanvas" >
                <div class="navbar navbar-default" role="navigation"> <!--//Navbar -->
                  <div class="side_nav"><!--nav-collapse-->
                    <ul class="nav navbar-nav">
                      <li><a href="index.html"> Home </a> </li>
                      <li class="active"><a href="#"> About Us <span class="caret"></span> </a>
                        <ul class="dropdown-menu">
                          <li><a href="introduction.html">Introduction</a></li>
                          <li><a href="mission.html">Our Mission and Vision </a></li>
                          <li><a href="message.html">Director's Message</a></li>
                          <li><a href="team.html">Our Team</a></li>
                          <li><a href="why_page.html">Why Educare ?</a></li>
                        </ul>
                      </li>
                      <li><a href="#">Services <span class="caret"></span></a>
                        <ul class="dropdown-menu">
                          <li><a href="translation.html">Translation</a></li>
                          <li><a href="test_preparation.html">Test Preparation </a></li>
                          <li><a href="special_course.html">Special Course for Individual</a></li>
                          <li><a href="japan_study.html">Japan Study Counselling</a></li>
                          <li><a href="home_stay.html">Home Stay Program</a></li>
                        </ul>
                      </li>
                      <li><a href="#">Study Abroad <span class="caret"></span></a>
                        <ul class="dropdown-menu">
                          <li><a href="translation.html">Translation</a></li>
                          <li><a href="test_preparation.html">Test Preparation </a></li>
                          <li><a href="special_course.html">Special Course for Individual</a></li>
                          <li><a href="japan_study.html">Japan Study Counselling</a></li>
                          <li><a href="home_stay.html">Home Stay Program</a></li>
                        </ul>
                      </li>
                      <li><a href="gallery_listing.html">Gallery </a></li>
                      <li><a href="gallery_listing.html">Downloads </a></li>
                      <li><a href="contact_us.html"> contact us </a> </li>
                    </ul>
                  </div>
                  <!--/.nav-collapse --> 
                </div>
                <!--//Navbar --> 
              </div>
              <!--offcanvas-->
              <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"  onclick="openNav3();openOffcanvas()"> <span class="toggle-bar"></span> </button>
            </div>
          </div>
        </div>
      </div>
    </div>
    <!-- ./navigation-bar--> 
  </div>
  <!-- ./pav-mainnav--> 
</header>
<!--../header ends-->

<section class="sub_page-banner">
  <figure class="full_banner"> <img src="images/contact_banner.png"> </figure>
  <div class="container breadCrumb_container">
    <div class="breadCrumbNav">
      <h1 class="breadCrumb_title">Contact us </h1>
    </div>
  </div>
</section>
<section class="inner_page section_wrapper">
  <div class="container inner_content">
    <div class="row contact_content">
      <div  class="col col-sm-8 col-xs-6 contact_form">
        <h3>Get In Touch with Us !</h3>
        <form role="form">
          <div class="row">
            <div class="form-group col-sm-6 col-xs-12">
              <input id="name" class="form-control" type="text" placeholder="Name:" title="Type your full name">
            </div>
            <div class="form-group col-sm-6 col-xs-12">
              <input id="phone" class="form-control" type="tel" placeholder="Phone:" title="Type your phone no">
            </div>
            <div class="form-group col-sm-6 col-xs-12">
              <input id="email" class="form-control" type="email" placeholder="E-Mail:" title="Type your email">
            </div>
            <div class="form-group col-sm-6 col-xs-12">
              <input id="email" class="form-control" type="text" placeholder="Subject:" title="Type your Subject">
            </div>
            <div class="form-group col-sm-12 col-xs-12">
              <textarea id="message" class="form-control" placeholder="Message..."></textarea>
            </div>
            <div class="form-group col-sm-12 col-xs-12">
              <input class="btn_submit btn " type="submit" value="Submit">
            </div>
          </div>
        </form>
      </div>
      <div class="col col-sm-4 col-xs-6 contact_info">
        <address>
        <figure class="icon"> <i class="fa fa-map-marker" aria-hidden="true"></i></figure>
        <div class="details"> <span>Rajmarg Chauraha, Butwal, Nepal </span> <span>(TVS Showroom Building)</span> </div>
        </address>
        <address>
        <figure class="icon"><i class="fa fa-phone" aria-hidden="true"></i></figure>
        <div class="details"> <span>+977-71-547163 </span> </div>
        </address>
        <address>
        <figure class="icon"> <i class="fa fa-mobile" aria-hidden="true"></i> </figure>
        <div class="details"> <span>+977-9857032709,  9804421509</span> </div>
        </address>
        <address>
        <figure class="icon"> <i class="fa fa-envelope" aria-hidden="true"></i> </figure>
        <div class="details"> <a href="info@educare.edu.np">info@educare.edu.np</a> </div>
        </address>
        <address>
        <figure class="icon"> <i class="fa fa-globe" aria-hidden="true"></i> </figure>
        <div class="details"> <a href="https://www.educare.edu.np/">https://www.educare.edu.np/</a> </div>
        </address>
      </div>
    </div>
  </div>
  <div class="map">
    <div class="gmap_canvas">
      <iframe src="https://www.google.com/maps/embed?pb=!1m23!1m12!1m3!1d56527.838423082474!2d83.43005035528338!3d27.6867069967474!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!4m8!3e0!4m5!1s0x399686879aa7be65%3A0xce71a46dac556cfe!2sEducare+Institute+Pvt.+Ltd.%2C+Rajmarg+Chauraha%2C+Butwal%2C+Nepal!3m2!1d27.6867114!2d83.46507!4m0!5e0!3m2!1sen!2sus!4v1562752097839!5m2!1sen!2sus" width="600" height="450" frameborder="0" style="border:0" allowfullscreen></iframe>
    </div>
  </div>
</section>

<!--// footer starts-->
<footer class="footer">
  <div class="container">
    <div class="footer_contact_info">
      <div class="row">
        <div class="col col-sm-4 fadeInUp animatable">
          <address>
          <figure class="icon"><i class="fa fa-envelope"></i> </figure>
          <div class="details">
            <h4>Email</h4>
            <a href="mailto: info@educare.edu.np" class="linkto"> info@educare.edu.np</a> </div>
          </address>
        </div>
        <div class="col col-sm-4 fadeInUp animatable">
          <address>
          <figure class="icon"> <i class="fa fa-map-marker" aria-hidden="true"></i> </figure>
          <div class="details">
            <h4> Address</h4>
            <span> Rajmarg Chauraha, Butwal, Nepal 
            (TVS Showroom Building)</span> </div>
          </address>
        </div>
        <div class="col col-sm-4 fadeInUp animatable">
          <address>
          <figure class="icon"><i class="fa fa-volume-control-phone" aria-hidden="true"></i></figure>
          <div class="details">
            <h4> Phone</h4>
            <span> +977-71-547163</span> </div>
          </address>
        </div>
        <div class="col-sm-12">
          <div class="footer-social-icons">
            <ul>
              <li><a href="#"><i class="fa fa-facebook" aria-hidden="true"></i> </a></li>
              <li><a href="#"><i class="fa fa-twitter" aria-hidden="true"></i> </a></li>
              <li><a href="#"><i class="fa fa-linkedin" aria-hidden="true"></i></a></li>
              <li><a href="#"><i class="fa fa-instagram" aria-hidden="true"></i></a></li>
              <li><a href="#"><i class="fa fa-youtube-play" aria-hidden="true"></i></a></li>
            </ul>
          </div>
        </div>
      </div>
    </div>
    <div class="copyright">
      <div class="row">
        <div class="col-sm-offset-2 col-sm-8">
          <p class="flt_lft">Copyright© 2018, DCT Web agency. All rights Reserved. </p>
          <p class="flt_rht">Powered by : <a href="http://www.brevincreation.com" target="_blank" class="company_link">Brevin Creation</a> </p>
        </div>
      </div>
    </div>
  </div>
</footer>

<!--// footer ends-->

<section class="back_top"><!--//back to top scroll-->
  <div class="container">
    <div id="back-top" style="display: block;"> <a href="#top" title="Go to top"><i class="fa fa-long-arrow-up" aria-hidden="true"></i> </a> </div>
  </div>
</section>
<!--//back to top scroll--> 

<script src="js/jquery-1.9.1.min.js"></script> 
<script src="js/fixed-nav.js"></script> 
<script src="js/jquery.js"></script> 
<script src="js/bootstrap.js"></script> 
<script src="js/Push up jquery.js"></script> 
<script type="text/javascript" src="js/equalheight.js"></script> 
<script type="text/javascript" src="SmartDroddownMenus/smart_dropdown.js"></script> 
<script type="text/javascript" src="js/annimatable_jquery.js"></script> 
<script type="text/javascript" src="js/side_nav.js"></script>
</body>
</html>
