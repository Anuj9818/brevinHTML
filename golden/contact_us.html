<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta name="description" content="">
<meta name="author" content="">
<link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon">
<title>Home | Contact us</title>
<link rel="stylesheet" type="text/css" href="css/bootstrap.css" />
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.2.0/css/all.css" integrity="sha384-hWVjflwFxL6sNzntih27bfxkr27PmbbK/iSvJ+a4+0owXq79v+lsFkW54bOGbiDQ" crossorigin="anonymous">
<link rel="stylesheet" type="text/css" href="FontAwesome 4.1.0/css/font-awesome.css"/>
<link rel="stylesheet" type="text/css" href="css/style.css" />
<link rel="stylesheet" type="text/css" href="css/responsive.css" />
<link href="css/owl.carousel.css" rel="stylesheet">
<link href="css/jquery.bxslider.css" rel="stylesheet">

<!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
<!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
<!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->

<!--logo slider Starts -->

<!--logo slider ends -->
<link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i,800,800i" rel="stylesheet">
<link href="https://fonts.googleapis.com/css?family=Montserrat:300,400,500,600,700,800" rel="stylesheet">
<link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700" rel="stylesheet">
</head>

<body>
<header class="header"><!--//.header starts-->
  <div class="top-bar">
    <div class="container">
      <ul class="social-icons">
        <li><a href="#"><i class="fa fa-facebook" aria-hidden="true"></i> </a></li>
        <li><a href="#"><i class="fa fa-twitter" aria-hidden="true"></i> </a></li>
        <li><a href="#"><i class="fa fa-linkedin" aria-hidden="true"> </i></a></li>
        <li><a href="#"><i class="fa fa-google-plus" aria-hidden="true"></i> </a></li>
      </ul>
    </div>
  </div>
  <div id="pav-mainnav">
    <div class="navigation-bar">
      <div class="container">
        <div class="row">
          <div class="col-sm-4">
            <figure class="logo_holder"> <a href="index.html"><img src="images/main-logo.png"></a> </figure>
          </div>
          <div class="col-sm-8 nav_col">
            <div class="navbar" role="navigation"> 
              <!--navbar-->
              <div class="navbar-header navbar-inverse">
                <h4>Menu</h4>
                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar"></span> <span class="icon-bar"></span> <span class="icon-bar"></span> </button>
              </div>
              <div class="collapse navbar-collapse"><!--nav-collapse-->
                
                <ul class="nav navbar-nav">
                  <li><a href="index.html">Home </a></li>
                  <li><a href="about_us.html" > About Us </a></li>
                  <li><a href="our_services.html"> Our Services </a> </li>
                  <li><a href="our_clients.html"> Our Clients </a></li>
                  <li><a href="gallery.html"> Gallery</a></li>
                  <li class="active"><a href="contact_us.html" > Contact Us</a></li>
                </ul>
              </div>
            </div>
          </div>
          <!-- /.nav-collapse --> 
        </div>
      </div>
    </div>
  </div>
</header>
<!--//.header ends--> 

<!--.//.MAIN SLIDER STARTS-->
<section class="banner">
  <figure class="about-banner"> <img src="images/contact-banner.jpg"  alt="" title=""> </figure>
  <div class="breadCrumbNav">
    <div class="container">
      <h2 class="page_title">Contact us</h2>
    </div>
  </div>
</section>
<!--.//.MAIN SLIDER ENDS--> 

<!-- //CONTACT CONTENT STARTS-->
<section class="content-holder contact_content page_holder">
  <div class="container">
  <div class="row">
    <div class="col-sm-offset-3 col-sm-6">
      <div class="txt_wrapper">
        <h1>KEEP IN TOUCH</h1>
        <p>Professional cleaning & renovation services in kathmandu, 
          For more information in our service please contact us by phone or email.</p>
      </div>
    </div>
    </div>
    
    <!--<div class="row">
      <div class="col-sm-4 col-xs-12">
        <address>
        <figure class="icon"> <i class="fa fa-map-marker" aria-hidden="true"></i> </figure>
        <div class="details"> <span>Lalitpur,Kupondole,</span> <span> Kathmandu </span> </div>
        </address>
      </div>
      <div class="col-sm-4 col-xs-12">
        <address>
        <figure class="icon"> <i class="fa fa-phone" aria-hidden="true"></i> </figure>
        <div class="details"> <span> <a href="tel:+015555039" class="linkto">+015555039 </a> </span> </div>
        </address>
      </div>
      <div class="col-sm-4 col-xs-12">
        <address>
        <figure class="icon"> <i class="fa fa-envelope" aria-hidden="true"></i> </figure>
        <div class="details"> <span> <a href="mailto:info@goldentriangle.com.np" class="linkto">info@goldentriangle.com.np</a> </span> </div>
        </address>
      </div>
    </div>-->
    <div class="map">
      <div class="gmap_canvas">
        <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3531.9042008141796!2d85.31781121511978!3d27.72024398278663!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x39eb191b9f0ca651%3A0x720f23d97bcbcd91!2sTAG+nepal+Treks+%26+Expedition+Pvt+Ltd!5e0!3m2!1sen!2sus!4v1544172177689" width="600" height="450" frameborder="0" style="border:0" allowfullscreen></iframe>
      </div>
    </div>
    <div class="row">
      <form role="form" class="contact_form">
        <div class="col-sm-12">
          <h1> Get In Touch</h1>
          <p>We have helped thousands of clients reach their full potential, let use help you! </p>
        </div>
        <div class="col-sm-6">
          <div class="form-group ">
            <input id="name" class="form-control" type="text" placeholder="Your Name:" title="Type your full name">
          </div>
          <div class="form-group">
            <input id="email" class="form-control" type="email" placeholder="E-Mail:" title="Type your email">
          </div>
          <div class="form-group ">
            <input id="phone" class="form-control" type="text" placeholder="Phone:" title="Type your pnone no">
          </div>
          <div class="form-group ">
            <input id="subject" class="form-control" type="text" placeholder="Subject:" title="Type your Subject">
          </div>
        </div>
        <div class="col-sm-6">
          <div class="form-group ">
            <textarea id="message" class="form-control" rows="6" placeholder="Your Message..."></textarea>
          </div>
          <div class="form-group ">
            <input class="btn-submit" type="submit" value="Submit" data-loading-text="Loading...">
          </div>
        </div>
      </form>
    </div>
  </div>
</section>
<!-- //CONTACT CONTENT ENDS-->

<section class="content-holder footer">
  <div class="container">
    <div class="row inner_content">
      <div class="col-sm-3 col-xs-12">
        <h4 class="title">Info</h4>
        <ul class="page_links sameheight3">
          <li><a href="index.html">Home </a></li>
          <li><a href="about_us.html">About Us </a></li>
          <li><a href="our_services.html">Our Services </a></li>
          <li><a href="our_clients.html">Our Clients </a></li>
          <li><a href="news.html">News & Updates </a></li>
          <li><a href="gallery.html">GAllery </a></li>
          <li><a href="contact_us.html">Contact US </a></li>
        </ul>
      </div>
      <div class="col-md-offset-1 col-md-3 col-sm-offset-1 col-sm-4 col-xs-12">
        <h4 class="title">Contact Us</h4>
        <div class="contact_wrapper sameheight3">
          <address>
          <figure class="icon"> <i class="fa fa-map-marker" aria-hidden="true"></i> </figure>
          <div class="details"> <span> Lalitpur,Kupondole<br>
            Kathmandu </span> </div>
          </address>
          <address>
          <figure class="icon"> <i class="fa fa-phone" aria-hidden="true"></i> </figure>
          <div class="details"> <span> <a href="tel:015555039" class="linkto">015555039 </a> </span> </div>
          </address>
          <address>
          <figure class="icon"> <i class="fa fa-envelope" aria-hidden="true"></i> </figure>
          <div class="details"> <span> <a href="mailto:info@goldentriangle.com.np" class="linkto">info@goldentriangle.com.np</a> </span> </div>
          </address>
        </div>
      </div>
      <div class="col-md-offset-1 col-md-3 col-sm-4 col-xs-12">
        <h4 class="title">Get Social</h4>
        <ul class="social-icons sameheight3">
          <li><a href="#" data-toggle="tooltip" data-placement="top" title="" data-original-title="Facebook"><i class="fa fa-facebook" aria-hidden="true"></i> </a></li>
          <li><a href="#" data-toggle="tooltip" data-placement="top" title="" data-original-title="Twitter"><i class="fa fa-twitter" aria-hidden="true"></i> </a></li>
          <li><a href="#" data-toggle="tooltip" data-placement="top" title="" data-original-title="Linkedin"><i class="fa fa-linkedin" aria-hidden="true"> </i></a></li>
          <li><a href="#" data-toggle="tooltip" data-placement="top" title="" data-original-title="Google-plus"><i class="fa fa-google-plus" aria-hidden="true"></i> </a></li>
        </ul>
      </div>
    </div>
    <div class="copyright">
      <p>© Copyright © 2018. Golden Triangle. All Right Reserved.</p>
    </div>
  </div>
</section>
<section class="back-top">
  <div id="back-top" class="uparrow" style="display: block;"> <a href="#top" title="Go to the top of the page"> <i class="fa fa-angle-up"></i> </a> </div>
</section>
</body>
<script src="js/jquery-1.9.1.min.js"></script>
<!--./testimonial slider starts-->

<!--./testimonial slider ends-->
<script src="js/fixed-nav.js"></script>
<script src="js/jquery.js"></script>
<script src="js/bootstrap.js"></script>
<script src="js/Push-up.js"></script>

<!--..//SAME HEIGHT JQUERY STARTS-->
<script type="text/javascript" src="js/sameheight.js">
</script>

<!--..//SOCIAL ICON HOVER TOOLTIP STARTS-->
<script>
$(document).ready(function(){
    $('[data-toggle="tooltip"]').tooltip();   
});
</script>
</html>
